app/server/app/routes/data.js

```diff
@@ -9,6 +9,62 @@ const StreamingService = require("../utilities/streamingService");
 
 // mapping to dynamically build GET/POST endpoints and queries
 const mapping = {
+  assessmentUnits: {
+    tableName: "assessment_units",
+    idColumn: "id",
+    columns: [
+      { name: "id", alias: "id" },
+      { name: "assessmentunitid", alias: "assessmentUnitId" },
+      { name: "assessmentunitname", alias: "assessmentUnitName" },
+      { name: "assessmentunitstate", alias: "assessmentUnitState" },
+      { name: "locationdescription", alias: "locationDescription" },
+      { name: "locationtext", alias: "locationText" },
+      { name: "locationtypecode", alias: "locationTypeCode" },
+      { name: "objectid", alias: "objectId" },
```

Comment:
Do we need to return the `objectId`, or even ETL it into the database? When we first talked about adding it, I'd assumed it would only be used on the MV side of things to optimize query offsets.

Reply:
I don't think we need `objectid`. We can probably remove these from the app and etl.